src/main/webapp/app/entities/exam-users-attendance-check.model.ts

```diff
@@ -0,0 +1,11 @@
+import { BaseEntity } from 'app/shared/model/base-entity';
+
+export class ExamUserAttendanceCheck implements BaseEntity {
```

Comment:
Is this a DTO? Then it should be named so.

Reply:
Yes its a DTO, i renamed it. :+1: